lib/spatial_features/has_spatial_features/feature_import.rb

```diff
@@ -54,6 +54,10 @@ def update_features!(skip_invalid: false, allow_blank: false, **options)
       if skip_invalid
         Rails.logger.warn "Error updating #{self.class} #{self.id}. #{e.message}"
         return nil
+      elsif ENCODING_ERROR.match?(e.message)
+        raise ImportEncodingError,
+              "One or more features you are trying to import has text encoded in an un-supported format (#{e.message})",
```

Comment:
Do we have any actionable suggestions at this point? e.g. Make sure the spatial file is saved with UTF-8 encoding.

Reply:
This could theoretically be triggered by any importer, although it was Shapefiles that actually caused us to get here.  I'm not sure what options users of Shapefiles (or even were Shapefiles come from -- a stork I think?)

UTF-8 is definitely safe.  Maybe something like "Make sure your mapping program exports data in UTF-8 format" ?

I would definitely defer to you for language